src/main/java/randoop/generation/AbstractGenerator.java

```diff
@@ -331,6 +331,13 @@ public void createAndClassifySequences() {
           outErrorSeqs.add(eSeq);
         } else {
           outRegressionSeqs.add(eSeq);
+
+          if (GenInputsAbstract.enable_bloodhound) {
+            // Increment the number of times this last operation was successfully invoked
+            // as the final method of a new sequence.
+            TypedOperation lastOperation = eSeq.sequence.getOperation();
+            newRegressionTestHook(lastOperation);
```

Comment:
Both the invocation and the definition of this method are guarded by `enable_bloodhound`.  That doesn't achieve abstraction.  At the least, the call here should be unconditional, and the hook method should take the sequence rather than the operation.

Ideally, when enable_bloodhound is true, Randoop could set a field to an object, like TypedOperationSelector, that does the work, and enable_bloodhound wouldn't be used except at Randoop startup.